drivers/ethernet/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ zephyr_sources_ifdef(CONFIG_ETH_DW		eth_dw.c)
 zephyr_sources_ifdef(CONFIG_ETH_E1000		eth_e1000.c)
 zephyr_sources_ifdef(CONFIG_ETH_ENC28J60	eth_enc28j60.c)
 zephyr_sources_ifdef(CONFIG_ETH_MCUX		eth_mcux.c)
+zephyr_sources_ifdef(CONFIG_ETH_SMSC911X	eth_smsc911x.c)
```

Comment:
Merge this patch with previous one. Driver comes all file at once: code, Kconfig and built directives

Reply:
Hmm, didn't notice we had it like that. Done.